ci-operator/step-registry/workers-scale/workers-scale-commands.sh

```diff
@@ -89,6 +95,7 @@ if [[ $worker_count_num -gt 0 ]]; then
             waitForReady $worker_count_num
         else
             scaleMachineSets $worker_count_num
+            crdApprove
```

Comment:
s/crdApprove/csrApprove/g

`csrApprove` should be invoked, whenever `waitForReady $worker_count_num` is invoked.